### PR TITLE
Adding support to control main wifi as well

### PR DESCRIPTION
**Pull request by janbernhardt**
I like to turn off my wifi during the night. So I copied and modified your guest wifi code so that another switch is now available to turn on/off the primary wifi.

**Breaking Change:**
`switch.fritzbox_guestwifi` -> `switch.fritzbox_guest_wifi`

### CODE REVIEW

**Comment by janbernhardt** [comment]
Successfully tested on Fritz!Box 7362 SL, but should work on others as well.

**Comment by mammuth** [comment]
Cool, might be useful for some people :+1:

File: custom_components/fritzbox_tools/switch.py [hunk 1]
```diff
@@ -506,3 +507,130 @@ def available(self) -> bool:
             return False
         else:
             return True
+
+class FritzBoxWifiSwitch(SwitchDevice):
+    """Defines a FRITZ!Box Tools Wifi switch."""
+
+    name = "FRITZ!Box Wifi"
```
Comment (AaronDavidSchneider) [inline custom_components/fritzbox_tools/switch.py]: for name we would then need a dictionary defined in `const.py`:

```py
DEF_NETWORK_NAMES = {"1": "FRITZ!Box Wifi", "2": "FRITZ!Box Wifi (5GHz)","3":"Guest Wifi"}
```
->
```py
name = DEF_NETWORK_NAMES[self.network]
```

and something similar in entity_id (maybe using slugify)

**Comment by mammuth** [comment]
@janbernhardt thanks for your contribution! If you find the time, it'd be cool if you'd checkout @AaronDavidSchneider comments and ideas! 😊

**Comment by janbernhardt** [comment]
@AaronDavidSchneider great ideas! How can I apply and test all your suggested changes?
Do you have another clone of this project including your suggested  changes? Or what is the general procedure here?

**Comment by AaronDavidSchneider** [comment]
> @AaronDavidSchneider great ideas! How can I apply and test all your suggested changes?
> Do you have another clone of this project including your suggested changes? Or what is the general procedure here?

Hey @janbernhardt: feel free to implement them yourself 😄
If you want me to implement these changes I can do so.

**Comment by janbernhardt** [comment]
Hi, I refactored my code to use one class supporting multiple wifi switches.
Please have a review and send me feedback,

Since your master branch has changed in the mean time I had to do a merge commit.
If your would like to avoid this, I could also rebase my code changes based on your current master and force my previous commits to be overwritten. However since my clone was public before, I would like to avoid this. WDYT?

Please be aware of a breaking change that comes with my commit. Switch name has changed from  fritzbox_guestwifi to fritzbox_guest_wifi (actually mapping the documentation in the readme.md).
If you don't like this I could also change my entity_id replace filter to simply remove whitespaces instead of inserting underscores. WDYT?

**Comment by AaronDavidSchneider** [comment]
Hey @janbernhardt,
this looks good to me! Works on my setup.
@mammuth: I would suggest that this PR is merged before #59 and merge the new changes in there. What do you think?

**Comment by mammuth** [comment]
> @mammuth: I would suggest that this PR is merged before #59 and merge the new changes in there. What do you think?

Since we need to wait until HA 0.105 with #59 your suggestion makes sense.

**Comment by mammuth** [comment]
Thanks for you contribution @janbernhardt! I just published 1.6.0 to HACS with your changes 😊

**Pull request merged**